api/review/review-controller.ts

```diff
@@ -0,0 +1,205 @@
+import { Router, Request, Response } from "express";
+import {
+  getAllReviews,
+  createReview,
+  getSpecificReviewDetails,
+} from "./review-service";
+import { SERVER_ERROR, UNAUTHORIZED_ERROR } from "../utils/errors";
+import { authenticate } from "../auth/auth";
+import { validateRequest } from "../utils/validator";
+import { reviewSchema } from "./review-model";
+async function handleCreateReview(req: Request, res: Response) {
+  try {
+    const userRole = res.locals.user.role || "";
+    if (userRole !== "employer")
+      return res.status(401).send(UNAUTHORIZED_ERROR);
+    await createReview(req.body, res.locals.user.email);
+    // TODO send mail
+
+    res.status(200).send();
+  } catch (err) {
+    console.log(err);
+    if (err.code) {
+      res.status(err.code).json(err.message);
```

Comment:
add return in all such places
```suggestion
      return res.status(err.code).json(err.message);
```